android/java/org/chromium/chrome/browser/customtabs/BraveAccountCustomTabActivity.java

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.customtabs;
+
+import static androidx.browser.customtabs.CustomTabsIntent.COLOR_SCHEME_DARK;
+import static androidx.browser.customtabs.CustomTabsIntent.COLOR_SCHEME_LIGHT;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.net.Uri;
+import android.provider.Browser;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.view.View;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.browser.customtabs.CustomTabsIntent;
+import androidx.core.content.ContextCompat;
+
+import org.chromium.base.IntentUtils;
+import org.chromium.build.annotations.NullMarked;
+import org.chromium.build.annotations.Nullable;
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.tab.Tab;
+import org.chromium.components.browser_ui.styles.SemanticColorUtils;
+import org.chromium.ui.util.ColorUtils;
+
+@NullMarked
+public class BraveAccountCustomTabActivity extends CustomTabActivity {
+    private static final int CLOSE_BUTTON_MARGIN_DP = 16;
+    private static final int HEADER_HEIGHT_DP = 64;
+    private static final int TITLE_TEXT_SIZE_SP = 18;
+
+    @Override
+    public void performPostInflationStartup() {
+        super.performPostInflationStartup();
+
+        // Hide the toolbar container
+        View toolbarContainer = findViewById(R.id.toolbar_container);
+        if (toolbarContainer != null) toolbarContainer.setVisibility(View.GONE);
+
+        TextView title = new TextView(this);
+        title.setText(R.string.prefs_section_brave_account);
+        title.setTextSize(TypedValue.COMPLEX_UNIT_SP, TITLE_TEXT_SIZE_SP);
+        title.setTextColor(ContextCompat.getColor(this, R.color.schemes_on_surface_variant));
```

Comment:
I wonder why presubmit didn't issue a warning, `setTextSize`/`setTextColor` must not be used, the preferred way is `setTextAppearance` with style at `android/java/res/values/brave_styles.xml`. 

Please see https://github.com/brave/brave-core/commit/5bb879b937c27082d8041fc9f03f37239a3bb09e as a reference.